src/resolution_functions/instrument_data/arcs.yaml

```diff
@@ -8,10 +8,8 @@ version:
             d_sample_detector: 3.0                  # Distance (x2) from sample to detector (m)
             aperture_width: 0.1751         # Width of aperture at moderator face (m)
             theta: -13.75                   # Angle beamline makes with moderator face (degrees)
-            q_size: 100
```

Comment:
Should these data changes be in this PR?

Reply:
They are there because I noticed while writing documentation that some of the values were not being used. ig it would be better to separate them, but I don't think it's much of an issue.

Reply:
Ok, in this case let's leave them here.